tests/rt_cascading.erl

```diff
@@ -128,14 +136,45 @@ simple_test_() ->
             ?assertEqual(Bin, maybe_eventually_exists(State#simple_state.middle, ?bucket, Bin)),
             ?assertEqual(Bin, maybe_eventually_exists(State#simple_state.ending, ?bucket, Bin))
         end},
+
         {"check pendings", fun() ->
             wait_until_pending_count_zero([State#simple_state.middle,
                                            State#simple_state.beginning,
                                            State#simple_state.ending])
         end}
-    ] end}}.
-
-big_circle_test_() ->
+    ],
+    lists:foreach(fun({Name, Eval}) ->
+        lager:info("===== simple: ~s =====", [Name]),
+        Eval()
+    end, Tests).
+
+big_circle() ->
+    State = big_circle_setup(),
```

Comment:
This ought to be called `Nodes` since the state variable is really a list of Nodes.